libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1125,6 +1125,8 @@ void AP_OSD_Screen::draw_distance(uint8_t x, uint8_t y, float distance)
         } else {
             fmt = "%4.0f%c";
         }
+    } else if (distance_scaled < 10.0f) {
+        fmt = "% 3.1f%c";
```

Comment:
no space after %

Reply:
actually provides better format....its supposed to be right justified with a %x.y% format, but isnt....this provides some space...its supposed to be a standard padding in c++ print formats...one space works, but it really needs two "%  3.1% but that doesnt add the second leading space for some reason